examples/github_app.ml

```diff
@@ -46,10 +46,17 @@ let pipeline ~app () =
   Github.Api.Repo.ci_refs ~staleness:(Duration.of_day 90) repo
   |> Current.list_iter (module Github.Api.Commit) @@ fun head ->
   let src = Git.fetch (Current.map Github.Api.Commit.id head) in
-  Docker.build ~pool ~pull:false ~dockerfile (`Git src)
-  |> Current.state
-  |> Current.map github_status_of_state
-  |> Github.Api.Commit.set_status head "ocurrent"
+
+  let build = 
+    Docker.build ~pool ~pull:false ~dockerfile (`Git src) 
+    |> Current.state 
+  in
+
+  Current.all [
+     (* build |> Current.map github_status_of_state |> Github.Api.Commit.set_status head "ocurrent"
+     * ; *)
+      build |> Current.map github_check_run_status_of_state |> Github.Api.CheckRun.set_status head (Github.App.get_app_id app)
```

Comment:
Alternative to the commented out build that uses `Github.Api.Commit.set_status`

Reply:
I think the previous layout was clearer. You're never going to want to set both, so `Current.all` seems wrong.